packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -18,12 +20,17 @@ export default async function processDeployment({
 }: any) {
   const { warn, log } = createOutput({ debug });
   let bar: Progress | null = null;
+  const useHttp2 = semver.satisfies(pkg.version, '>10');
```

Comment:
I don't think this is correct. You need to look at the current version of node running. Probably use `process.versions.node`.

Reply:
`process.version`

Reply:
Oh crap. Right